ccd-definition/CaseField/CareSupervision/caseField.json

```diff
@@ -1530,7 +1530,7 @@
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "CARE_SUPERVISION_EPO",
     "ID": "migrateToMoWarning",
-    "Label": "# Check you're registered for an account\n\nAfter 14 September 2020, only registered users can create cases on this system.\n\nAsk your team administrator to [check if you're registered](https://manage-org.platform.hmcts.net/organisation).",
+    "Label": "# Register for an account\n\nYou cannot start an online application until you’re fully registered.\n\nRegister at [https://manage-org.platform.hmcts.net/organisation](https://manage-org.platform.hmcts.net/organisation).",
```

Comment:
la solicitor cannot register in MO on his own, so showing link can be misleading. Organisation admin can invite users to organisation

Reply:
spoke to Collette, Jason etc - removing this.